docs_app/content/guide/operators.md

```diff
@@ -121,7 +121,7 @@ There are operators for different purposes, and they may be categorized as: crea
 
 For a complete overview, see the [references page](/api).
 
-### Creation Operators
+### <a id="creation-operators-list"></a>Creation Operators ###
```

Comment:
Is the added `###` really needed? Seem to be invalid markdown to me?

Reply:
those are not needed. I have removed them.